packages/ActionMenu/README.md

```diff
@@ -0,0 +1,46 @@
+# ActionMenu
+
+This component is used to display a kebab menu which will contain a list of menu options. The menu is dismissible if you click outside the dropdown.
```

Comment:
This component will not only be used with a kebab menu, it must be possible to use it with a button or other.

Example: 

<img width="263" alt="Capture d’écran, le 2023-03-02 à 13 09 37" src="https://user-images.githubusercontent.com/2379339/222514913-b61cd6ab-50bb-4586-acb6-5ef8adfff412.png">


<img width="270" alt="Capture d’écran, le 2023-03-02 à 13 10 50" src="https://user-images.githubusercontent.com/2379339/222515378-bb02aa5c-04c8-452a-bd61-6ecb8a7cd235.png">